notify/README.md

```diff
@@ -0,0 +1,101 @@
+# Collection - notify
+
+## API Endpoints
+
+|            Route             |                  Description                  |
+| :--------------------------: | :-------------------------------------------: |
+| [POST /notify](#post-notify) | send notification to your specified fcm token |
+|                              |
+
+## POST /notify
+
+    Send notification to your specified device.
+
+- **Params**  
+  None
+- **Query**
+  None
+- **Body**
+  - Attributes
+    - **title** (required, string): notification title
+    - **body** (required, string): notification message
```

Comment:
can we specify allowed character types & maximum lengths considering FCM allows only 4 kb of data

Reply:
yes we will be giving it in frontend side

Reply:
We need to have it from backend as well

Reply:
sure